### DISCUSSION TITLE
Possibile to find clearAll() using Ray App

### DISCUSSION THREAD

**Discussion by lloricode** [spatie/ray#506] [Ideas]
**Describe the bug**
This is not a bug, I just notice if we forgot to remove `ray()->clearAll();` it will difficult to find it, since its not included in Ray App
![Screenshot from 2021-07-10 14-03-54](https://user-images.githubusercontent.com/8251344/125153620-b0530f00-e187-11eb-98c9-25830d7dd619.png)

**Versions**
Ray version (you can see this in "About Ray"): 1.15.0

You can use `composer show` to get the version numbers of:
- spatie/ray package version: 1.28.0
- spatie/laravel-ray package version (if applicable): 1.23.0

PHP version: 7.4.18
Laravel version (if applicable): 8.49.2

**To Reproduce**
1. add only `ray()->clearAll()` anywhere in code base
2. run the code
3. Check Ray App its empty on list
![Screenshot from 2021-07-10 14-03-54](https://user-images.githubusercontent.com/8251344/125153712-425b1780-e188-11eb-9829-30d218f5d642.png)

**Expected behavior**
It will also show it, in case to forgot to remove, so able to easy locate

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. macOS] : Linuxmint
 - Version [e.g. 22] : Uma 20.2
 
 **Additional context**
also `ray()->clearScreen();`, and other non visual on Ray App

Thank you

**Comment by freekmurze** [comment]
We're aware of this problem, and we'll ship some improvements around this in the coming months.

**Reply by lloricode** [reply]
that is great!, thanks @freekmurze

**Comment by lloricode** [comment]
this is solved by https://github.com/spatie/x-ray

**Comment by kktsvetkov** [comment]
How about sending a log message acknowledging the clearAll? This way after cleaning you will always know what called it and from where:

> Everything cleared
> TestScript.php:78
